src/bundle/bundle.js

```diff
@@ -0,0 +1,18 @@
+const core = require("@actions/core");
+const exec = require("@actions/exec");
+
+class Bundle {
+  async publish(path, channel) {
+    core.exportVariable("CHARMCRAFT_AUTH", core.getInput("credentials"));
+    process.chdir(path);
+    await exec.exec("juju-bundle", [
```

Comment:
Just a note for @DomFleischmann that we need to think carefully about using this, and what the succession plan is for that tool now that Ken has likely stopped maintaining it.

Reply:
Yeah, I actually considered just removing it and hacking out the parts needed in the action instead, but I don't know really how much black magic it actually does in the background, other than rendering the bundle yaml and publishing it.

Reply:
Yeh maybe worth tracking in a separate issue